pango/sys/src/lib.rs

```diff
@@ -865,13 +887,19 @@ impl ::std::fmt::Debug for PangoGlyphInfo {
 pub struct PangoGlyphItem {
     pub item: *mut PangoItem,
     pub glyphs: *mut PangoGlyphString,
+    pub y_offset: c_int,
+    pub start_x_offset: c_int,
+    pub end_x_offset: c_int,
```

Comment:
Are those fine? don't we need different ones for < & > 1.50?

Reply:
That means that we were never supposed to allocate this ourselves and that it shouldn't be boxed-inline. No other changes needed apart from that.